src/olympia/addons/indexers.py

```diff
@@ -260,9 +248,12 @@ def extract_compatibility_info(cls, obj, version_obj):
                     # alone to leave the API representation intact.
                     max_ = version_int('9999')
             else:
-                # Fake wide compatibility for search tools and personas.
-                min_, max_ = 0, version_int('9999')
-                min_human, max_human = None, None
+                # Fake wide compatibility for add-ons with no info. We don't
+                # want to reindex every time a new version of the app is
+                # released, so we directly index a super high version as the
+                # max.
+                min_human, max_human = '1.0', '9999'
```

Comment:
Can we have these magic numbers as constants?  Makes me nervous seeing them hardcoded everywhere.